wurst/systems/spawns/BushSelection.wurst

```diff
@@ -0,0 +1,23 @@
+package BushSelection
+import HashMap
+import RegisterEvents
+
+let MAX_DIST = 300
+let MAX_DIST_SQ = compiletime(MAX_DIST * MAX_DIST)
+
+let map = new HashMap<int, int>()

```

Comment:
Will this work for sub-classes?

Reply:
Good point, probably not

Reply:
The loop for giving the items also looks like it won't try every slot. If a troll only has 1 slot open it will only check the first slot of the bush for items instead of going until it selects an item.

Reply:
Good catch @Aluvian-Sprinter